torch/testing/_internal/common_utils.py

```diff
@@ -2161,6 +2170,12 @@ def assertEqual(
         # Hide this function from `pytest`'s traceback
         __tracebackhide__ = True
 
+        if isinstance(x, torch.Tensor) and (x.is_mps):
+            x = x.to('cpu')
+
+        if isinstance(y, torch.Tensor) and (y.is_mps):
+            y = y.to('cpu')
```

Comment:
Probably don't want to permanently do this..

Reply:
This is going to need a comment -- is this necessary because we don't support enough operations to compare two mps tensors for equality?

Reply:
Added a comment.